spec/services/invite_provider_user_spec.rb

```diff
@@ -31,6 +31,13 @@
       it 'a provider user is created' do
         expect(ProviderUser.find_by_email_address('test+invite_provider_user@example.com')).not_to be_nil
       end
+
+      it 'queues an email' do
+        message_delivery = instance_double(ActionMailer::MessageDelivery, deliver_later: nil)
+        allow(ProviderMailer).to receive(:account_created).and_return(message_delivery)
```

Comment:
I forgot to submit this review! Now I see @tijmenb's suggestion which sounds better

Reply:
:+1: I've added to the system spec. I changed the asserts for the service to be less implementation dependent I think - I kept them here too because I want to test the positive and negative cases and the system spec only does the former.